api/v1alpha1/worker_types.go

```diff
@@ -215,6 +215,10 @@ type TargetWorkerDeploymentVersion struct {
 	// Only set when Status is VersionStatusRamping.
 	// +optional
 	RampingSince *metav1.Time `json:"rampingSince"`
+
+	// RampLastModifiedAt is the time when the ramp percentage was last changed.
```

Comment:
nit: If the ramping version changes, but the percentage stays the same (ie. 5% -> B, and then 5% -> C), would `RampLastModifiedAt` change? I'm assuming yes the ramp was modified, because that's what makes sense and what the variable name implies. Maybe it's obvious, but the comment only says "percentage" which could be misleading

Reply:
```suggestion
	// RampLastModifiedAt is the time when the ramp percentage was last changed for the target version.
```